package-lock.json

```diff
@@ -0,0 +1,5096 @@
+{
```

Comment:
Could you remove this lock file please? I'll add an .npmrc file to prevent these being generated in a later PR.